src/export/export-chakra.ts

```diff
@@ -117,64 +123,96 @@ const processTextStyles = (
 
   // Replace the font family in each text style with the corresponding token
   Object.keys(updatedTextStyles).forEach((key) => {
-    const value = updatedTextStyles[key];
-    const fontFamily = value.fontFamily;
-    if (typeof fontFamily === 'string') {
-      const token = getFontFamilyToken(fontFamily);
-      if (token) {
-        value.fontFamily = token;
+    if (typeof updatedTextStyles[key] !== 'string') {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      const value = updatedTextStyles[key];
+      const fontFamily = value.fontFamily;
+      if (typeof fontFamily === 'string') {
+        const token = getFontFamilyToken(fontFamily);
+        if (token) {
+          value.fontFamily = token;
+        }
       }
     }
   });
 
   return updatedTextStyles;
 };
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const reshapeDesignTokens = (tokens: any): ChakraTokens => {
```

Comment:
Should we add some documentation to this function to make it clear what it's doing?

Reply:
Also let's get rid of the `any` here. Is the `any` only necessary here because our `Tokens` type is inaccurate?

Reply:
This one is likely due to a lack of typescript knowledge on my part.
When replacing `any` with `Tokens` I get a lot of type errors that I'm not 100% sure how to fix.

Reply:
It looks like the TypeScript errors are because our type definition for `Tokens` doesn't match the actual object 🤔 

Here's my attempt at creating a simplistic but accurate definition:

```

```

Note that `Dictionary<T>` is just a shortcut to say "an object with `string`-type keys and `T`-type values".

I'll leave it up to you to do this in a smarter way (e.g. updating the existing `Breakpoint`, `Radii`, etc. types instead of getting rid of them) and to update the rest of the code if it throws a fit about the new type definition 😊 Reach out to me if you have any issues and we can pair on it.

Reply:
Thought about this more and it may be more complex than even I expected 🤔 Sit tight and we'll discuss the best approach in our 1:1 😊